processor/src/chiplets/aux_trace/mod.rs

```diff
@@ -7,7 +9,31 @@ mod bus;
 pub(crate) use bus::{ChipletLookup, ChipletsBus, ChipletsBusRow};
 
 mod virtual_table;
+
+const NUM_HEADER_ALPHAS: usize = 4;
+
```

Comment:
Let's create "CONSTANTS" section for this.

Reply:
Done!